console/src/layout/slice.ts

```diff
@@ -191,6 +191,14 @@ export const { actions, reducer } = createSlice({
       const { location, name, tab } = layout;
       let key = layout.key;
 
+      // Handle modal layouts separately
+      if (location === "modal") {
```

Comment:
Why was this added? Layout behavior seems like it's been pretty consistent for a while now.